app/services/flow/flows/provider_capital.rb

```diff
@@ -40,8 +40,23 @@ class ProviderCapital < FlowSteps
               :offline_accounts
             end
           end,
+          check_answers: ->(app) { app.checking_non_passported_means? ? :check_capital_answers : :check_passported_answers }, # TODO: Change to add_other_vehicles
```

Comment:
TODO: handle, ticket and/or remove?

Reply:
Removed in a subsequent rebase - it was a question for me that I answered but forgot to remove!